ui/printer.py

```diff
@@ -117,6 +118,7 @@ def exit_printer():
         i.set_callback("KEY_LEFT", exit_printer)
         i.set_callback("KEY_ENTER", exit_printer)
         i.listen()
+    fit_image_to_screen(image, o)
```

Comment:
here, `image` is returned but you never assign it to `image` variable used by the GP.